src/main/java/com/iabtcf/decoder/ConsentParser.java

```diff
@@ -0,0 +1,311 @@
+     */
+    boolean getUseNonStandardStacks();
+
+    /**
+     * The TCF Policies designates certain Features as "special" which means a CMP must afford the
+     * user a means to opt in to their use. These "Special Features" are published and numerically
+     * identified in the Global Vendor List separately from normal Features.
+     *
+     * @since 2.0
+     * @return the Special Features the Vendor may utilize when performing some declared Purposes
+     *         processing.
+     */
+    IntIterable getSpecialFeatureOptIns();
+
+    /**
+     * The Purpose’s transparency requirements are met for each Purpose on the legal basis of
+     * legitimate interest and the user has not exercised their "Right to Object" to that Purpose.
+     * 
+     * By default or if the user has exercised their "Right to Object" to a Purpose, the
+     * corresponding identifier for that Purpose is set to false.
+     * 
+     * @since 2.0
+     * @return The purpose identifiers for which the legal basis of legitimate interest are
+     *         established.
+     */
+    IntIterable getPurposesLITransparency();
+
+    /**
+     * CMPs can use the PublisherCC field to indicate the legal jurisdiction the publisher is under
+     * to help vendors determine whether the vendor needs consent for Purpose 1.
+     * 
+     * In a globally-scoped TC string, this field must always have a value of false. When a CMP
+     * encounters a globally-scoped TC String with PurposeOneTreatment set to true then it is
+     * considered invalid and the CMP must discard it and re-establish transparency and consent.
+     *
+     * @since 2.0
+     * @return true if Purpose 1 was NOT disclosed; false otherwise.
+     */
+    boolean getPurposeOneTreatment();
+
+    /**
+     * The country code of the country that determines legislation of reference. Commonly, this
+     * corresponds to the country in which the publisher’s business entity is established.
+     *
+     * @since 2.0
+     * @return ISO 3166-1 alpha-2 country code
+     */
+    String getPublisherCC();
+
+    /**
+     * If a user exercises their "Right To Object" to a vendor’s processing based on a legitimate
+     * interest.
+     *
+     * @since 2.0
+     * @return vendor identifiers that can process this user based on legitimate interest
+     */
+    IntIterable getVendorLegitimateInterest();
+
+    /**
+     * The restrictions of a vendor's data processing by a publisher within the context of the users
+     * trafficking their digital property.
+     * 
+     * @since 2.0
+     * @return the list of publisher restrictions.
+     */
+    List<PublisherRestriction> getPublisherRestrictions();
+
+    /**
+     * Part of the OOB segments expressing that a Vendor is using legal bases outside of the TCF to
+     * process personal data.
+     * 
+     * @since 2.0
+     * @return A list of Vendors that the publisher allows to use out-of-band legal bases.
+     */
+    IntIterable getAllowedVendors();
```

Comment:
Including an OOB word in the method name would make this more apparent that this is a special list of vendors outside the normal means of consent.

Reply:
where else would it come from?

Reply:
It's just a matter of convenience. This being a field that conveys information separate from the normal means of consent, it would be nice to have the method name hint at that.

Reply:
i would agree if there was an allowedvendor field in another segment. i don't see there being confusion as is.